css/style.css

```diff
@@ -0,0 +1,348 @@
+
+.work-title {
+  width: 295px;
+  height: 36px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 6px 0;
+}
+
+.clients {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+}
+
+h4 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.img-clients {
+  padding: 5px;
+}
+
+.highlights {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 4px;
+}
+
+.project-detail {
+  width: 100px;
+  height: 40px;
+  border-width: 1px;
+  border-color: #396df2;
+  border-radius: 0.5em;
+  cursor: pointer;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.04em;
+  color: #396df2;
+  background: #fff;
+  margin-bottom: 10px;
+}
+
+.project-detail:hover {
+  background: #396df2;
+  color: #fff;
+}
+
+hr {
+  border: 0;
+  clear: both;
+  display: block;
+  background-color: #dfe1e6;
+  height: 1px;
+  margin: 1rem 3rem;
+}
+
+div .taggs {
+  float: right;
+  width: 40px;
+  height: 20px;
+  padding-bottom: 20px;
+}
+
+.tag1 {
+  width: 50px;
+  height: 20px;
+  margin-right: 8px;
+  border: none;
+  border-radius: 6px;
+  margin-bottom: 20px;
+  color: #396df2;
+}
+
+.tag2 {
+  width: 40px;
+  height: 20px;
+  margin-right: 8px;
+  border: none;
+  border-radius: 6px;
+  color: #396df2;
+}
+
+.tag3 {
+  width: 80px;
+  height: 20px;
+  margin-right: 8px;
+  border: none;
+  border-radius: 6px;
+  color: #396df2;
+}
+
+/* ///////// About myself//////// */
+
+.about-myself {
+  box-sizing: border-box;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.about-myself-title {
+  width: 327px;
+  height: 101px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-bottom: 1px;
+  padding-left: 0.5rem;
+}
+
+.myself-content {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding: 0 1rem 0 0.5rem;
+}
+
+.links {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin-bottom: 1px;
+  padding-left: 0.5rem;
+}
+
+.socials {
+  display: flex;
+  list-style: none;
+  padding: 0 1px;
+}
+
+.socials li {
+  padding-left: 16px;
+}
+
+.resume {
+  width: 130px;
+  height: 44px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  border-width: 1px;
+  border-color: #396df2;
+  color: #396df2;
+  border-radius: 0.5em;
+  background: #fff;
+  cursor: pointer;
+  margin-left: 0.5rem;
+}
+
+.resume:hover {
+  background: #396df2;
```

Comment:
Kindly fix the button hover color with the color code provided on the design. Also please add "Pressed" state.
<img width="1091" alt="Screenshot 2023-01-13 at 12 15 47 AM" src="https://user-images.githubusercontent.com/25887253/212186376-aefd1cd2-8a63-40d7-b1ca-35996b6736f8.png">